### PR TITLE
refactor(recipe): Added parts for recipe model

### LINKED ISSUE #15: Divide recipe model into a more structured way
Refactoring the recipe model into more structure, as discussed [here](https://github.com/Mindener-Landfrauen/Brauser.RecipeEditor/issues/14#issuecomment-1504691970)

1. General information
2. Brewing process
3. Fermentation and maturing

### PR DESCRIPTION
**Pull request by tdankert**
Started to divide the recipe into logical parts.

* Filled GeneralInformation
* Created `BrewingProcess.cs` and `FermentationAndMaturing.cs`

@Johannes123123412341412412412412414 maybe you could shed some light onto this. I'm not sure, which things do belong to the Brewing and what belongs to the fermentation.

Closes #15

### CODE REVIEW

**Comment by n-develop** [comment]
Is this still a draft on purpose?

File: Brauser.Common/Recipe/DryHop.cs [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+/// While importing from MaischeMalzUndMehr.de it may be required to fill StartTime and Duration
+/// </summary>
 public class DryHop
 {
     public Guid Id { get; set; }
     public int? Order { get; set; }
     public string? Variety { get; set; }
     public string? Quantity { get; set; }
-}
+    public TimeSpan StartTime { get; set; }
```
Comment (n-develop) [inline Brauser.Common/Recipe/DryHop.cs:12]: If it is a "Time", it should be of type `DateTimeOffset`, shouldn't it?

Comment (n-develop) [reply Brauser.Common/Recipe/DryHop.cs:12]: Or is this more like "after X minutes" dry hop should be added?

**Review by n-develop — APPROVED** [review]
I like the concept of splitting the model up to make it better readable. 👍

File: Brauser.Common/Recipe/DryHop.cs (hunk 1 above)
Comment (tdankert) [reply Brauser.Common/Recipe/DryHop.cs:12]: The second one. After some time you have to add stuff

**Pull request merged**